JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -770,6 +777,15 @@ - (void)messagesInputToolbar:(MEGAInputToolbar *)toolbar didPressJoinButton:(UIB
 }
 
 - (void)messagesInputToolbar:(MEGAInputToolbar *)toolbar didChangeToState:(InputToolbarState)state {
+    switch (state) {
```

Comment:
This block of code could have gone in the same method in our subclass, `MessagesViewController`. We try to put there as much code as possible, but as we are going to replace this vendor, it is ok for me here.